visivo/models/model.py

```diff
@@ -1,12 +1,53 @@
-from typing import Optional
-from pydantic import Field, validator
+from typing import Any, List, Optional, Union
+from typing_extensions import Annotated
+from pydantic import Field, Discriminator, Tag
 from visivo.models.base.named_model import NamedModel
 from visivo.models.base.parent_model import ParentModel
-from visivo.models.target import DefaultTarget, Target
-from .base.base_model import generate_ref_field
+from visivo.models.target import DefaultTarget, Target, TypeEnum
+from .base.base_model import RefString, generate_ref_field
+import pandas
+import re
 
 
-class Model(NamedModel, ParentModel):
+class Model(NamedModel):
+    pass
+
+
+class RunModel(Model):
+    cmds: List[str] = Field(description="The sql used to generate your base data")
+
+    @property
+    def sql(self):
+        return f"select * from {self.table_name}"
```

Comment:
I like this. So the default is that they get the whole file but if they want to also run sqlite on top of the output they can?

Reply:
Hadn't thought of that, currently you get the whole thing that is in the csv.

Reply:
We could take that as a second pass.  My thought is that the script you are running would only return items you care about for the base model level.